content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,97 @@
+
+> "For instance, on the planet Earth, man had always assumed that he was more intelligent than dolphins because he had achieved so much—the wheel, New York, wars and so on—whilst all the dolphins had ever done was muck about in the water having a good time. But conversely, the dolphins had always believed that they were far more intelligent than man—for precisely the same reasons."  
+> - Douglas Adams *from* Hitchhiker's Guide to the Galaxy, 1979
+
+### Humans are different to other animals
+
+That’s true. Humans have a capacity for complex problem solving that is not present in other animals. But why should a set of unique abilities or characteristics equate to superiority and give us the right to do what we like to those deemed inferior?
+
+> Until recently it was considered that our capacity for abstract thought made us unique. However, “there is growing literature indicating that animals are capable of causal reasoning and abstraction”  
+> [Decision-Making From the Animal Perspective: Bridging Ecology and Subjective Cognition (2019)](https://www.frontiersin.org/articles/10.3389/fevo.2019.00164/full)  
+
+Difference and uniqueness are not synonymous with superiority. 
+
+### The flawed reasoning of Superiority and Domination
+
+> For any X and Y, if X is superior (or morally superior) to Y, then X is morally justified in subordinating Y.
+
+This logic is used to claim that humans are superior to, and may dominate, non-humans *due to some difference* that is elevated (by humans) in importance. However, without the "assumptions  that *humans are superior* to (at least some) nonhumans, and that *superiority justifies subordination*, all one has is some difference" - Karen J. Warren *from* The Power and the Promise of Ecological Feminism *in* Environmental Ethics.
+
+This creates what Warren refers to as an *oppressive conceptual framework*. An example is patriarchal thinking which falsely justifies the subordination of women by men based on nothing more than difference. Warren further claims that this logic of domination, and any conceptual framework which gives rise to it, "must be abolished  in order to make possible a meaningful notion of difference..."
+
+We need to move away from the oppressive conceptual framework that maintains *sameness* and only extends the moral community to those who are similar to us by application of some self-determined criteria. Warren refers to this as *arrogant perception*, resulting in a false notion of superiority which gives rise to the claim that those like us deserve moral consideration and others don't.
+
+An alternative *loving perception* "presupposes and maintains difference - a distinction between the self and other, between human and at least some nonhumans - in such a way that perception of the other as other *is* an expression of love for one who/which is recognised at the outset as independent, dissimilar, different."
+
+
+
+### A Thought Experiment
```

Comment:
Can we add a short version of this to the quick responses? As I think it kicks ass.

Reply:
I suppose... should bperhaps to a 'besides ...' after the 'Even if....' response